### PR TITLE
Making changes based of specification review

### PR DESCRIPTION
**Pull request by sampankumar**
__Short description outlining key changes/additions__
Making changes based of specification review
__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/acsp/controllers/AcspController.java [hunk 1]
```diff
@@ -91,15 +84,16 @@ public ResponseEntity<Object> getAcspData(@PathVariable(TRANSACTION_ID_KEY) Stri
...
+    public ResponseEntity<Object> checkHasApplication(@PathVariable("acsp_application_id") String acspId,
+                                                      HttpServletRequest request){
         LOGGER.info("received request to check for user applications");
+        String requestId = request.getHeader(ApiSdkManager.getEricPassthroughTokenHeader());
         return acspService.getAcspApplicationStatus(acspId, requestId);
     }
 
-    @DeleteMapping("/acsp-api/user/{acsp_id}/application")
-    public ResponseEntity<Object> deleteApplication(@PathVariable("acsp_id") String acspId) {
+    @DeleteMapping("/acsp-api/user/{acsp_application_id}/application")
```
Comment (swalke-ch) [inline src/main/java/uk/gov/companieshouse/acsp/controllers/AcspController.java:95]: I don't think we are using this endpoint, If we are not planning to use it in future then we can remove it.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sardhani-ch — APPROVED** [review]

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Pull request merged**